.github/workflows/main.yml

```diff
@@ -37,6 +37,17 @@ jobs:
       - uses: actions/setup-python@v2
         with:
           python-version: '3.8'
+      - name: Install V
```

Comment:
Any chance of making binary distributions? I'll watch the amount of time it adds to CI.

Reply:
https://github.com/adsharma/py2many/pull/390/checks?check_run_id=3013635796 took 24 mins and this commit took 27 mins. Not sure what the run-to-run variation is, but 3 mins extra isn't the end of the world. But these things can add up over time.

Reply:
The runtime variation is more likely caused by GitHub itself, as https://github.com/adsharma/py2many/pull/390/checks?check_run_id=3015808449 also took 27m.

I don't like using the weekly binary distributions as since V is still evolving at a fast rate, they tend to lag. One of V's selling points is fast compilation and I don't really think it's much of an issue, although I can still use the weekly builds if you want.

Reply:
Takes 11 secs + clone time on my machine. So ok for now.